packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -29,6 +29,14 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // enable image provider domains and device sizes
+  images: {
+    domains: ['cm.jss.localhost'],
+    loader: 'custom',
+    path: 'https://cm.jss.localhost',
+    deviceSizes: [100, 300],
```

Comment:
Add, please, some comment and describe that these numbers tied to Sitecore config values